storage-api/src/java/org/apache/hadoop/hive/common/AcidConstants.java

```diff
@@ -40,6 +40,10 @@
   public static final String VISIBILITY_PREFIX = "_v";
   public static final Pattern VISIBILITY_PATTERN = Pattern.compile(VISIBILITY_PREFIX + "\\d+");
 
+  public static final String V_TABLE_SUFFIX = ".v";
```

Comment:
Could we use a variable name which describes these tables better?
Like: SOFT_DELETE_PATH_SUFFIX, SOFT_DELETE_TABLE_PATTERN, SOFT_DELETE_TABLE?

Reply:
@pvary, should we keep`versioned=true` as tableProperty or you prefer `soft_delete=true` ?